README.md

```diff
@@ -141,6 +143,8 @@ Command options for *serve*-command:
 | --liveness-probe   | The exposed path (default is /alive) for probes to check liveness |
 | --check-migrations | Check if all migrations were applied before starting application |
 | --webhook-url      | If specified, webhooks will be sent to this url |
+| --pycharm-host     | The host of the pycharm debug server |  
+| --pycharm-port     | The port of the pycharm debug server. This is only used in combination with the '--pycharm-host' option |  
```

Comment:
why don't you put both of these parameters combined in one option, i.e. [host]:[port]?

Reply:
I was thinking about that. But then we have to parse them back out again because they are used separately in the code. That means we have to be very clear that an option like "--pycharm-address" must have the format {host}:{port} and the code has to handle errors from parsing (no ":", port not an `int`, etc.) which can otherwise (as it is now) be done implicitly by the argparser.